app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -76,45 +104,74 @@ class AamvaStateMaintenanceWindow
         # ... and 8:30 am. to 8:35 am ET.
         { cron: '30 8 * * *', duration_minutes: 5 },
-        #  Sundays from 9 am. to 10:30 am. ET...
-        { cron: '0 9 * * Sun', duration_minutes: 90 },
-        # ...and 5 am to 5:45 am ET on 2nd Sunday of month.
-        { cron: '0 5 * * Sun#2', duration_minutes: 45 },
       ],
       'MT' => [
-        # Monthly maintenance occurs first Sunday of each month
-        # from 12:00 am to 6:00 am (Mountain Time zone).
-        { cron: '0 2 * * Sun#1', duration_minutes: 6 * 60 },
+        # Third Saturday of odd numbered months from 12:00 am to 6:00 am ET
+        # MW FIXME: I need to test if this is valid, or if the month has to be 1,3,5,7,9,11
+        { cron: '0 2 * /2 Sat#3', duration_minutes: 6 * 60 },
       ],
       'NC' => [
         # Daily, Midnight to 7:00 am. ET.
         { cron: '0 0 * * *', duration_minutes: 7 * 60 },
-        # Sundays from 5am. till Noon
-        { cron: '0 5 * * Sun', duration_minutes: 7 * 60 },
       ],
-      # NM: "Sunday mornings." (not modeling; too vague)
+      'ND' => [
+        # Wednesday around 7:30 pm to 7:35 pm ET
+        { cron: '30 19 * * Wed', duration_minutes: 5 },
+        # 3rd Sunday of month, 5 minutes anytime between midnight and noon.
+      ],
+      'NM' => [
+        # Sundays 8:00 am. to noon ET.
+        { cron: '0 8 * * Sun', duration_minutes: 4 * 60 },
+      ],
+      'NV' => [
+        # Tuesdays to Sundays: 2:00 am. to 3:15 am. ET
+        # MW FIXME: This wraps around, does Tue-Sun get parsed correctly?
+        { cron: '0 2 * * Tue-Sun', duration_minutes: 1.25 * 60 },
+      ],
       'NY' => [
         # Sunday maintenance 8 pm. to 9 pm. ET.
         { cron: '0 20 * * Sun', duration_minutes: 60 },
       ],
+      'OH' => [
+        # Daily 4:00 am. to 4:30 am. ET
+        { cron: '0 4 * * *', duration_minutes: 30 },
+      ],
+      'OR' => [
+        # Sunday 7:30 am. to 9:00 am. ET.
+        { cron: '30 7 * * Sun', duration_minutes: 1.5 * 60 },
+      ],
       'PA' => [
-        # Sunday maintenance may occur, often between 5:30 am. & 7:00 am. ET
-        { cron: '30 5 * * Sun', duration_minutes: 90 },
+        # Sunday 5:00 am. to 7:00 am. ET.
+        { cron: '0 5 * * Sun', duration_minutes: 2 * 60 },
+      ],
+      'RI' => [
+        # Either 3rd or 4th Sunday of each month, 7:30 am. to 10:00 am. ET.
+        { cron: '30 7 * * Sun#3', duration_minutes: 2.5 * 60 },
+        { cron: '30 7 * * Sun#4', duration_minutes: 2.5 * 60 },
       ],
       'SC' => [
-        # Sunday maintenance from 7:00 pm. to 10:00 pm. ET.
-        { cron: '0 19 * * Sun', duration_minutes: 3 * 60 },
+        # Sunday 6:00 pm. to 10:00 pm. ET.
+        { cron: '0 18 * * Sun', duration_minutes: 4 * 60 },
+      ],
+      'TN' => [
+        # Last Sunday of every month from 11:00 pm Sunday to 2:00 am. Monday ET
+        { cron: '0 23 * * Sun#4', duration: 3 * 60 },
+        { cron: '0 23 * * Sun#5', duration: 3 * 60 },
```

Comment:
This is something cron can't model, so I just do the 4th and 5th.

What do you think of me doing a bit of date math and conditional assignment here? I've tended to view this as a static config file since it's initialized as a constant at application start, but there's no reason I can't do something like...

```ruby
if there_are_5_sundays_this_month
  { cron: '0 23 * * Sun#5', duration: 3 * 60 }
else
  { cron: '0 23 * * Sun#4', duration: 3 * 60 }
end
```

I also briefly tried `0 23 25-31 * Sun`, but it looks like that matches the 25th-31st of the month and also on every Sunday, rather than Sundays on the 25th-31st.

Reply:
Fixed in https://github.com/18F/identity-idp/pull/11725/files